### PR TITLE
[Team1x1] Fix connector

### LINKED ISSUE #2414: [team1x1] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes
**Describe the bug**
The site server has been closed
It has been transferred to another server
He became unresponsive

**To Reproduce**
* Name of connector:  https://team1x1.com/

* Failed to load resource: net::ERR_FILE_NOT_FOUND
index.html:61 Loading Frontend: frontend@classic-dark
index.html:76 [Deprecation] HTML Imports is deprecated and will be removed in M80, around February 2020. Please use ES modules instead. See https://www.chromestatus.com/features/5144752345317376 and https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade for more details.
loadClassicPolymer @ index.html:76
VRV.mjs:305 VRV (Crunchyroll) login failed! Error: No login credentials provided!
    at Promise.then.oauthRequest.method (VRV.mjs:407)
    at new Promise (<anonymous>)
    at VRVCrunchyroll._login (VRV.mjs:399)
    at VRVCrunchyroll._onSettingsChanged (VRV.mjs:303)
    at EventTarget.load (Settings.mjs:300)
    at async loadEngine (index.html:54)
    at async main (index.html:87)
(anonymous) @ VRV.mjs:305
VRV.mjs:305 VRV (HiDive) login failed! Error: No login credentials provided!
    at Promise.then.oauthRequest.method (VRV.mjs:407)
    at new Promise (<anonymous>)
    at VRVHiDive._login (VRV.mjs:399)
    at VRVHiDive._onSettingsChanged (VRV.mjs:303)
    at EventTarget.load (Settings.mjs:300)
    at async loadEngine (index.html:54)
    at async main (index.html:87)
(anonymous) @ VRV.mjs:305
VRV.mjs:305 VRV (Rooster Teeth) login failed! Error: No login credentials provided!
    at Promise.then.oauthRequest.method (VRV.mjs:407)
    at new Promise (<anonymous>)
    at VRVRoosterteeth._login (VRV.mjs:399)
    at VRVRoosterteeth._onSettingsChanged (VRV.mjs:303)
    at EventTarget.load (Settings.mjs:300)
    at async loadEngine (index.html:54)
    at async main (index.html:87)
(anonymous) @ VRV.mjs:305
api.vrv.co/core/index:1 Failed to load resource: the server responded with a status of 401 (Unauthorized)
VRV.mjs:310 VRV (HiDive) initialization failed! Error: oauth_error: The request has expired.
    at VRV.mjs:326
(anonymous) @ VRV.mjs:310
api.vrv.co/core/index:1 Failed to load resource: the server responded with a status of 401 (Unauthorized)
VRV.mjs:310 VRV (Rooster Teeth) initialization failed! Error: oauth_error: The request has expired.
    at VRV.mjs:326
(anonymous) @ VRV.mjs:310
Team1x1.mjs:45 TypeError: Cannot read property 'text' of undefined
    at Team1x1.mjs:37 Team1x1
(anonymous) @ Team1x1.mjs:45
Team1x1.mjs:45 TypeError: Cannot read property 'text' of undefined
    at Team1x1.mjs:37 Team1x1 {id: "team1x1", label: "Team X", tags: Array(2), isLocked: false, initialized: true, …}
(anonymous) @ Team1x1.mjs:45
Promise.catch (async)
_getMangaList @ Team1x1.mjs:44
(anonymous) @ Connector.mjs:105
Promise.then (async)
updateMangas @ Connector.mjs:104
onUpdateMangaListClick @ mangas.html:233
handler @ template-stamp.html:96
Connector.mjs:441 GET https://team1x1.com/manga/a-half-step-as-unusual/ 404
fetchDOM @ Connector.mjs:441
_getChapterList @ Team1x1.mjs:52
(anonymous) @ Manga.mjs:134
(anonymous) @ Manga.mjs:133
Promise.then (async)
_getUpdatedChaptersFromWebsite @ Manga.mjs:132
(anonymous) @ Manga.mjs:98
Promise.then (async)
getChapters @ Manga.mjs:94
onSelectedMangaChanged @ chapters.html:244
runObserverEffect @ property-effects.html:217
runEffectsForProperty @ property-effects.html:162
runEffects @ property-effects.html:128
_propertiesChanged @ property-effects.html:1662
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
__enableOrFlushClients @ property-effects.html:1555
_flushClients @ property-effects.html:1530
_propertiesChanged @ property-effects.html:1658
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
runNotifyEffects @ property-effects.html:261
_propertiesChanged @ property-effects.html:1665
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
set @ property-effects.html:1817
onMangaClicked @ mangas.html:263
handler @ template-stamp.html:96
(anonymous) @ templatize.html:149
Team1x1.mjs:66 Error: Failed to receive content from "https://team1x1.com/manga/a-half-step-as-unusual/" (status: 404) - 
    at Connector.mjs:454 EventTarget {connector: Team1x1, id: "/manga/a-half-step-as-unusual/", title: "A half step as unusual", status: "available", chapterCache: Array(0), …}
(anonymous) @ Team1x1.mjs:66
Promise.catch (async)
_getChapterList @ Team1x1.mjs:65
(anonymous) @ Manga.mjs:134
(anonymous) @ Manga.mjs:133
Promise.then (async)
_getUpdatedChaptersFromWebsite @ Manga.mjs:132
(anonymous) @ Manga.mjs:98
Promise.then (async)
getChapters @ Manga.mjs:94
onSelectedMangaChanged @ chapters.html:244
runObserverEffect @ property-effects.html:217
runEffectsForProperty @ property-effects.html:162
runEffects @ property-effects.html:128
_propertiesChanged @ property-effects.html:1662
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
__enableOrFlushClients @ property-effects.html:1555
_flushClients @ property-effects.html:1530
_propertiesChanged @ property-effects.html:1658
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
runNotifyEffects @ property-effects.html:261
_propertiesChanged @ property-effects.html:1665
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
set @ property-effects.html:1817
onMangaClicked @ mangas.html:263
handler @ template-stamp.html:96
(anonymous) @ templatize.html:149
Manga.mjs:153 _getUpdatedChaptersFromWebsite Error: Failed to receive content from "https://team1x1.com/manga/a-half-step-as-unusual/" (status: 404) - 
    at Connector.mjs:454
(anonymous) @ Manga.mjs:153
Promise.catch (async)
_getUpdatedChaptersFromWebsite @ Manga.mjs:151
(anonymous) @ Manga.mjs:98
Promise.then (async)
getChapters @ Manga.mjs:94
onSelectedMangaChanged @ chapters.html:244
runObserverEffect @ property-effects.html:217
runEffectsForProperty @ property-effects.html:162
runEffects @ property-effects.html:128
_propertiesChanged @ property-effects.html:1662
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
__enableOrFlushClients @ property-effects.html:1555
_flushClients @ property-effects.html:1530
_propertiesChanged @ property-effects.html:1658
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
runNotifyEffects @ property-effects.html:261
_propertiesChanged @ property-effects.html:1665
_flushProperties @ properties-changed.html:341
_flushProperties @ property-effects.html:1510
_invalidateProperties @ property-effects.html:1482
set @ property-effects.html:1817
onMangaClicked @ mangas.html:263
handler @ template-stamp.html:96
(anonymous) @ templatize.html:149
Team1x1.mjs:45 TypeError: Cannot read property 'text' of undefined
    at Team1x1.mjs:37 Team1x1 {id: "team1x1", label: "Team X", tags: Array(2), isLocked: false, initialized: true, …}
(anonymous) @ Team1x1.mjs:45
Promise.catch (async)
_getMangaList @ Team1x1.mjs:44
(anonymous) @ Connector.mjs:105
Promise.then (async)
updateMangas @ Connector.mjs:104
onUpdateMangaListClick @ mangas.html:233
handler @ template-stamp.html:96

**Screenshots**
kk
uj

**HakuNeko :**
 - Version [V.6.17]

### PR DESCRIPTION
**Pull request by Sheepux**
Fixes #2414

### CODE REVIEW

File: src/web/mjs/connectors/Team1x1.mjs [hunk 1]
```diff
@@ -10,74 +10,58 @@ export default class Team1x1 extends Connector {
...
-                    };
-                } );
-                if( index < mangaPageLinks.length - 1 ) {
-                    return this._getMangaListFromPages( mangaPageLinks, index + 1 )
-                        .then( mangas => mangaList.concat( mangas ) );
-                } else {
-                    return Promise.resolve( mangaList );
-                }
-            } );
+        let data = await this.fetchDOM( request, 'div#page-manga div.container div.last-post-manga div.thumb div.info h3 a', 5 )
```
Comment (manh21) [inline src/web/mjs/connectors/Team1x1.mjs]: Missing semicolon. Otherwise looking good.

**Review by manh21 — CHANGES_REQUESTED** [review]

**Review by manh21 — APPROVED** [review]

**Review by manh21 — APPROVED** [review]

**Comment by Sheepux** [comment]
Don't worry about the missing approval. It just means that you're currently not into the approved contributors: https://github.com/orgs/manga-download/people

File: src/web/mjs/connectors/Team1x1.mjs [hunk 2]
```diff
@@ -10,74 +10,58 @@ export default class Team1x1 extends Connector {
...
     }
 
-    _getPageList( manga, chapter, callback ) {
-        let script = `new Promise( resolve => resolve( PushManga.pages.original.map( p => p.page + '?token=' + p.token ) ) );`;
+    async _getPages(chapter) {
+        let script = `
+        new Promise(resolve => {
+            var interval = setInterval(function() {
+                if (typeof PushManga !== "undefined") {
+                    clearInterval(interval);
```
Comment (27136773) [inline src/web/mjs/connectors/Team1x1.mjs]: No need to clear interval, promises can only be resolve/reject once and browser window will be closed anyway.

File: src/web/mjs/connectors/Team1x1.mjs [hunk 3]
```diff
@@ -10,74 +10,58 @@ export default class Team1x1 extends Connector {
...
             } );
     }
 
-    _getPageList( manga, chapter, callback ) {
-        let script = `new Promise( resolve => resolve( PushManga.pages.original.map( p => p.page + '?token=' + p.token ) ) );`;
+    async _getPages(chapter) {
+        let script = `
+        new Promise(resolve => {
+            var interval = setInterval(function() {
+                if (typeof PushManga !== "undefined") {
```
Comment (27136773) [inline src/web/mjs/connectors/Team1x1.mjs]: Can be simplified to `if(PushManga)`

**Review by 27136773 — APPROVED** [review]
Looks good, minor improvements possible

**Pull request merged**